### PR TITLE
Dungeon Settlement Scene

### LINKED ISSUE #436: [DUNGEON] Dungeon scene
## 🚀 Feature

A new scene for the dungeon mode.
- A user can access this scene from the Overworld.
- The DungeonSettlement is shown in this scene.

From this scene the user can:
- Go back to Overworld
- Go to the Supercampaign scene to fight a battle (depends on #435)

### PR DESCRIPTION
**Pull request by tkz00**
Closes #436

## Motivation

A new Dungeon feature will be implemented, so the scenes for that must be added to the project, giving access to these functionalities.

## Summary of changes

Added two new scenes with no functionality other than navigation

## How has this been tested?

Check the new `Dungeon` and `DungeonSettlement` scenes can be traversed correctly.

### CODE REVIEW

File: client/Assets/Scripts/Campaign/CampaignLevelIndicator.cs
Comment (ncontinanza) [inline client/Assets/Scripts/Campaign/CampaignLevelIndicator.cs:1]: Watch out for formatting here!

File: client/Assets/Scripts/Campaign/CampaignLevelsManager.cs
Comment (ncontinanza) [inline client/Assets/Scripts/Campaign/CampaignLevelsManager.cs:1]: Watch out for formatting here!

File: client/Assets/Scripts/Campaign/CampaignManager.cs
Comment (ncontinanza) [inline client/Assets/Scripts/Campaign/CampaignManager.cs:1]: Watch out for formatting here!

File: client/Assets/Scripts/CampaignsMap/CampaignItem.cs
Comment (ncontinanza) [inline client/Assets/Scripts/CampaignsMap/CampaignItem.cs:1]: Watch out for formatting here!

File: client/Assets/Scripts/CampaignsMap/CampaignsMapManager.cs
Comment (ncontinanza) [inline client/Assets/Scripts/CampaignsMap/CampaignsMapManager.cs:1]: Watch out for formatting here!

File: client/Assets/Scripts/KalineTreeManager.cs
Comment (ncontinanza) [inline client/Assets/Scripts/KalineTreeManager.cs:1]: Watch out for formatting here!

File: client/Assets/Scripts/UnitList/UnitListManager.cs
Comment (ncontinanza) [inline client/Assets/Scripts/UnitList/UnitListManager.cs:1]: Watch out for formatting here!

**Review by ncontinanza — CHANGES_REQUESTED** [review]

File: client/Assets/Scripts/Dungeon/DungeonSettlementManager.cs
[the remark this answers is not in the document]
Comment (tkz00) [reply client/Assets/Scripts/Dungeon/DungeonSettlementManager.cs:1]: Nice catch, removed!

**Review by ncontinanza — APPROVED** [review]
LGTM!

**Review by lotuuu — CHANGES_REQUESTED** [review]
The inventory button is still there in the Dungeon scene but doesn't do anything! I say we just delete it

**Comment by tkz00** [comment]
> The inventory button is still there in the Dungeon scene but doesn't do anything! I say we just delete it

Right! Removed the button!

**Review by lotuuu — APPROVED** [review]

File: client/Assets/Scenes/DungeonSettlement.unity
Comment (ncontinanza) [inline client/Assets/Scenes/DungeonSettlement.unity:1]: Notice that in this scene there are two currency sprites overlapping:

image

**Review by ncontinanza — CHANGES_REQUESTED** [review]

File: client/Assets/Scenes/DungeonSettlement.unity
Comment (tkz00) [reply client/Assets/Scenes/DungeonSettlement.unity:1]: Bullseye! It's fixed

**Review by ncontinanza — APPROVED** [review]

**Review by lotuuu — APPROVED** [review]

**Pull request merged**